Project.toml

```
@@ -1,16 +1,25 @@
 name = "MakieRecipes"
 uuid = "3c562d8e-4afa-4f1c-99bf-ad54af2b207f"
-authors = ["Simon Danisch"]
-version = "0.1.0"
+authors = ["Simon Danisch", "Anshul Singhvi <anshulsinghvi@gmail.com>"]
+version = "0.1.0-DEV"
 
 [deps]
+AbstractPlotting = "537997a7-5e4e-5d89-9595-2241ea00577e"
+MakieLayout = "5a521ce4-ebb9-4793-b5b7-b334dfe8393c"
 RecipesBase = "3cdcf5f2-1ef4-517c-9805-6587b60abb01"
+RecipesPipeline = "01d81517-befc-4cb6-b9ec-a95719d0359c"
 
 [compat]
-julia = "1"
+julia = "1.1"
 
 [extras]
+Literate = "98b081ad-f1c9-55d3-8b20-4c87d4299306"
+MarketData = "945b72a4-3b13-509d-9b46-1525bb5c06de"
+OrdinaryDiffEq = "1dea7af3-3e70-54e6-95c3-0bf5283fa5ed"
+Phylo = "aea672f4-3940-5932-aa44-993d1c3ff149"
+StochasticDiffEq = "789caeaf-c7a9-5a7d-9973-96adeb23e2a0"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+TimeSeries = "9e3dc215-6440-5c97-bce1-76c03772f85e"
 
```

I think CairoMakie is also needed for tests.
```suggestion
CairoMakie = "13f3f980-e62b-5c42-98c6-ff1f3baf88f0"
```